### PR TITLE
Update Observability overview page for 8.2

### LINKED ISSUE #1806: [Request] Update Observability overview page
## Description

The Observability **Overview** page is changing in 8.2. We should update screenshots and references to the UI text in the [Observability overview page](https://www.elastic.co/guide/en/observability/current/observability-ui.html).

## Collaboration

The docs team will lead producing the content.

## Suggested Target Release

8.2.0

## Stakeholders

?

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Fixes #1806 

Updates the Observability overview page to reflect changes happening in 8.2.

### CODE REVIEW

File: docs/en/observability/observability-ui.asciidoc [hunk 1]
```diff
@@ -27,42 +27,41 @@ Additionally, the alerts chart keeps you informed of any issues that you may nee
...
-== Services and transactions
-
-The *APM* chart displays the total number of services running within your environment
-and the total number of transactions per minute that were captured by the Elastic APM
-agent instrumenting those services.
+[[view-alerts]]
+== Alerts
 
-To drill-down and view this APM data in the *APM app*, click *View in app*.
+To help keep you aware of potential issues in your environments, the *Alerts* feed
```
Comment (ollyhowell) [inline docs/en/observability/observability-ui.asciidoc:33]:
```suggestion
To help keep you aware of potential issues in your environments, the *Alerts* view
```
Is it a feed or a view (or something else)?

File: docs/en/observability/observability-ui.asciidoc [hunk 2]
```diff
@@ -27,42 +27,41 @@ Additionally, the alerts chart keeps you informed of any issues that you may nee
...
+provides a snapshot of the alerts occurring within the specified time frame. Displayed are the
```
Comment (ollyhowell) [inline docs/en/observability/observability-ui.asciidoc:34]: Depends on earlier comment but reads awkwardly to me - better to start 'This view displays the rule types...' or similar?

File: docs/en/observability/observability-ui.asciidoc [hunk 3]
```diff
@@ -71,54 +70,58 @@ possible memory leaks and whether disk resources need to be added or redistribut
...
+
+[float]
+[[view-services-and-transactions]]
+== Services
+
+The *Services* chart displays the total number of services running within your environment
+and the total number of transactions per minute that were captured by the Elastic APM
+agent instrumenting those services.
+
+To drill-down and view this APM data in the *{apm-app}*, click *Show service inventory*.
```
Comment (ollyhowell) [inline docs/en/observability/observability-ui.asciidoc:89]:
```suggestion
To drill down and view this APM data in the *{apm-app}*, click *Show service inventory*.
```
I think I got the review nudge to change "drill-down" to "drill down" in a previous PR. Not highlighted all occurences here.

**Review by ollyhowell — APPROVED** [review]
Minor comments aside, LGTM! 
Good to see the linting tool working away in the background (I'm guessing...)

File: docs/en/observability/observability-ui.asciidoc (hunk 2 above)
Comment (colleenmcginnis) [reply docs/en/observability/observability-ui.asciidoc:34]: Ah, I agree. This was existing text -- I just moved it up from the bottom of the page to reflect the order of charts in the UI. What about something like:

>The **Alerts** table provides a snapshot of alerts occurring within the specified time frame.
The table includes the alert status, when it was last updated, the reason for the alert, and more.
>
>To see more details about these alerts, click **Show alerts**. For more information,
see [Create rules](https://www.elastic.co/guide/en/observability/current/create-alerts.html).

File: docs/en/observability/observability-ui.asciidoc (hunk 3 above)
Comment (colleenmcginnis) [reply docs/en/observability/observability-ui.asciidoc:89]: This is something we could lint for in the future. 🙂

File: docs/en/observability/observability-ui.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/en/observability/observability-ui.asciidoc:33]: Maybe "table"?

**Pull request merged**